orgs/wagtail_hooks.py

```diff
@@ -138,6 +167,9 @@ def get_edit_handler(self):
         tabs = [
             ObjectList(self.basic_panels, heading=_('Basic information')),
         ]
+        # this is removed for non-superusers in OrganizationForm
+        tabs.append(ObjectList(self.superuser_panels, heading=_('Advanced settings')))
```

Comment:
I wonder if we could do one of the following:
1. hide this if a superorganization of this current one already is the instance org. Reason being, this might be confusing for some superusers since they might think it needs to be set for all organizations in the hierarchy, not just the root
2. instead of hiding the field in those cases, validate and ignore the setting if some ancestor of the org is already the instance org.